src/vs/workbench/contrib/issue/electron-sandbox/issueReporterService2.ts

```diff
@@ -104,70 +81,9 @@ export class IssueReporter2 extends BaseIssueReporterService {
 			this.setSourceOptions();
 			this.render();
 		});
-
-		// Keep all event listerns involving window and issue creation
-		this.previewButton.onDidClick(async () => {
-			this.delayedSubmit.trigger(async () => {
-				this.createIssue();
-			});
-		});
-
-		this.addEventListener('disableExtensions', 'click', () => {
-			this.issueFormService.reloadWithExtensionsDisabled();
-		});
-
-		this.addEventListener('extensionBugsLink', 'click', (e: Event) => {
-			const url = (<HTMLElement>e.target).innerText;
-			windowOpenNoOpener(url);
-		});
-
-		this.addEventListener('disableExtensions', 'keydown', (e: Event) => {
-			e.stopPropagation();
-			if ((e as KeyboardEvent).keyCode === 13 || (e as KeyboardEvent).keyCode === 32) {
-				this.issueFormService.reloadWithExtensionsDisabled();
-			}
-		});
-
-
-		// THIS IS THE MAIN IMPORTANT PART
-		this.window.document.onkeydown = async (e: KeyboardEvent) => {
-			const cmdOrCtrlKey = isMacintosh ? e.metaKey : e.ctrlKey;
-			// Cmd/Ctrl+Enter previews issue and closes window
-			if (cmdOrCtrlKey && e.key === 'Enter') {
-				this.delayedSubmit.trigger(async () => {
-					if (await this.createIssue()) {
-						this.close();
-					}
-				});
-			}
-
-			// Cmd/Ctrl + w closes issue window
-			if (cmdOrCtrlKey && e.key === 'w') {
-				e.stopPropagation();
-				e.preventDefault();
-
-				const issueTitle = (<HTMLInputElement>this.getElementById('issue-title'))!.value;
-				const { issueDescription } = this.issueReporterModel.getData();
-				if (!this.hasBeenSubmitted && (issueTitle || issueDescription)) {
-					// fire and forget
-					this.issueFormService.showConfirmCloseDialog();
-				} else {
-					this.close();
-				}
-			}
-
-			// With latest electron upgrade, cmd+a is no longer propagating correctly for inputs in this window on mac
-			// Manually perform the selection
-			if (isMacintosh) {
-				if (cmdOrCtrlKey && e.key === 'a' && e.target) {
-					if (isHTMLInputElement(e.target) || isHTMLTextAreaElement(e.target)) {
-						(<HTMLInputElement>e.target).select();
-					}
-				}
-			}
-		};
 	}
 
+	// Uses native host service instead.
```

Comment:
What's this comment mean?

Reply:
oo leftover from when i was differentiating functions between web and desktop